src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,83 @@
 import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  selectedTodo: Todo | null
+  onSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>;
```

Comment:
`onSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>` same is here, I think you can simplify it by `onSelectedTodo: (value: string) => void`

Reply:
> `onSelectedTodo: React.Dispatch<React.SetStateAction<Todo | null>>` same is here, I think you can simplify it by `onSelectedTodo: (value: string) => void`

That's ok, in this case, you can pass also a callback function as `setState` accepts it
https://www.freecodecamp.org/news/using-typescript-in-react-apps/#how-to-type-hooks